app/controllers/users_controller.rb

```diff
@@ -21,14 +21,19 @@ def update
       redirect_to admin_dashboard_index_path
     elsif current_user != nil
       current_user.update(user_params)
-      redirect_to dashboard_index_path
+      flash[:notice] = "Successfully updated your account information."
+      redirect_to account_edit_path
```

Comment:
Can we make this throw an error if something goes wrong?

Reply:
failure will be added!
I'm not going to be able to fix that nav bar tonight (or tomorrow morning) though.